requests_toolbelt/adapters/appengine.py

```diff
@@ -123,19 +161,29 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
             **response_kw)
 
 
-def monkeypatch():
+def monkeypatch(validate_certificate=True):
     """Sets up all Sessions to use AppEngineAdapter by default.
 
     If you don't want to deal with configuring your own Sessions,
     or if you use libraries that use requests directly (ie requests.post),
     then you may prefer to monkeypatch and auto-configure all Sessions.
+
+    .. warning: :
+
+        If ``validate_certificate`` is ``False``, certification validation will
+        effectively be disabled for all requests.
     """
     _check_version()
     # HACK: We should consider modifying urllib3 to support this cleanly,
     # so that we can set a module-level variable in the sessions module,
     # instead of overriding an imported HTTPAdapter as is done here.
-    sessions.HTTPAdapter = AppEngineAdapter
-    adapters.HTTPAdapter = AppEngineAdapter
+    if validate_certificate:
```

Comment:
Since there's other review feedback here, what if we do:

```py
adapter = AppEngineAdapter
if not validate_certificate:
    adapter = InsecureAppEngineAdapter
```